### PR TITLE
Link decorator fix for service messages (DLG-35204)

### PR DESCRIPTION
**Pull request by meafmira**

### CODE REVIEW

File: src/decorators/link.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * @flow
  */
 
-const pattern = /(?:\[(.+)\]\()?((?:https?|ftp):\/\/\S+)/ig;
+const pattern = /(?:\[(.+)\]\()?((?:https?|ftp):\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b(?:[-a-zA-Z0-9@:%_\+.~#?&\/\/=():!,\'\'\*]*))/ig;
```
Comment (nkt) [inline src/decorators/link.js]: nope =)
https://a.yandex

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dlghq/markdown",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (nkt) [inline package.json]: Update this from master

Comment (nkt) [reply package.json]: `npm version patch`

**Comment by meafmira** [comment]
@nkt updated

**Pull request merged**